src/app.js

```diff
@@ -0,0 +1,10 @@
+import { initStore } from './store/index.js';
+import { Home } from './template/index.js';
+
+const $app = document.getElementById('app');
+
+(async () => await initStore())()
+  .then(() => Home($app));
```

Comment:
```suggestion
initStore().then(() => Home($app));
```
이렇게 해도 된답니다! 굳이 async/await + 즉시실행 함수까지 할 필요는 없을 것 같아요~

Reply:
혹시 이렇게하면 성능면에서도 어떤 차이가 있는지 궁금합니다! @JunilHwang